src/main/resources/avro/common.avdl

```diff
@@ -18,33 +337,116 @@ enum Strand {
 }
 
 /**
-An abstraction for referring to a genomic position, in relation to some
-already known reference. For now, represents a genomic position as a reference
-name, a base number on that reference (0-based), and a flag to say if it's the
-forward or reverse strand that we're talking about.
+A `Position` is a side of a base pair in some already known sequence. A
+`Position` is represented by a sequence name or ID, a base number on that
+sequence (0-based), and a `Strand` to indicate the left or right side.
+
+For example, given the sequence "GTGG", the `Position` on that sequence at
+offset 1 in the forward orientation would be the left side of the T/A base pair.
+The base at this `Position` is "T". Alternately, for offset 1 in the reverse
+orientation, the `Position` would be the right side of the T/A base pair, and
+the base at the `Position` is "A".
+
+Offsets added to a `Position` are interpreted as reading along its strand;
```

Comment:
This sentence seems confusing given [line 36](https://github.com/ga4gh/schemas/pull/238/files#diff-f478e5ba9b38667091d5f5a7e1a0fc97R36) above.

The "offset" discussed here, which moves "right to left" (i.e. 5'-to-3') along the negative strand, sounds like it has the opposite behavior from the "positions" discussed previously (also referred to as "indices" on [L50](https://github.com/ga4gh/schemas/pull/238/files#diff-f478e5ba9b38667091d5f5a7e1a0fc97R50)) and the comment on the `position` field below ([L376](https://github.com/ga4gh/schemas/pull/238/files#diff-f478e5ba9b38667091d5f5a7e1a0fc97R376), which says it measures from "the start of the forward strand").

Am I misreading? Are there deliberately different semantics between {"indices", "positions"} vs. "offsets"?